src/schema/notifications.ts

```diff
@@ -254,4 +262,26 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       return { _: true };
     },
   },
+  Subscription: {
```

Comment:
This would be for the realtime capabilities, is it not something OneSignal already comes with perhaps that we can tap into?

Reply:
When we worked on the DR we discussed the different alternatives and agreed that since we have graphql subscriptions already it will be the easiest and most flexible to use. Also, OneSignal doesn't provide real-time directly as far as I'm aware.